pkg/catalogv2/git/download.go

```diff
@@ -19,13 +19,18 @@ import (
 const (
 	stateDir  = "management-state/git-repo"
 	staticDir = "/var/lib/rancher-data/local-catalogs/v2"
+	localDir  = "../rancher-data/local-catalogs/v2" // identical to helm.InternalCatalog
```

Comment:
I don't see how localDir is different from staticDir ? Can you explain the change of behaviour of Rancher when localDir is introduced ? instead of using staticDir ?